### ISSUE TITLE
Package manager has died crash on Android 6.0.1 devices

### ISSUE THREAD

**Issue by rpavliuk** [talsec/Free-RASP-Android#24]
**Describe the bug**
We have started to receive a crash on 20+ users after upgrading to the latest version of the library.

**To Reproduce**
Have no idea, because this crash occurs only on users' devices. However, the absence of the logs and keys in Crashlytics gives a hint that this crash occurs on app start.

**Expected behavior**
Working without crash 🙂 

**Please complete the following information:**
 - Devices: Galaxy Note4, Galaxy A7, Galaxy Grand Prime Plus, LG X Power. 80% are Samsung devices.
 - OS version: Android 6.0.1
 - Version of freeRASP: 8.0.4

**Additional context**
Here is a crash report from Crashlytics:

First part
```
Fatal Exception: java.lang.RuntimeException: Package manager has died
       at android.app.ApplicationPackageManager.queryIntentActivitiesAsUser(ApplicationPackageManager.java:843)
       at android.app.ApplicationPackageManager.queryIntentActivities(ApplicationPackageManager.java:807)
       at com.aheaditec.talsec.security.o1.a(SourceFile:153)
       at com.aheaditec.talsec.security.o1.a(SourceFile:141)
       at com.aheaditec.talsec.security.b.a(SourceFile:51)
       at com.aheaditec.talsec.security.o1.c(SourceFile:3)
       at com.aheaditec.talsec.security.o1.a(SourceFile:1)
       at com.aheaditec.talsec.security.a2.d(SourceFile:3)
       at java.lang.Thread.run(Thread.java:818)
```

Second part
```
Caused by android.os.DeadObjectException: Transaction failed on small parcel; remote process probably died
       at android.os.BinderProxy.transactNative(Binder.java)
       at android.os.BinderProxy.transact(Binder.java:503)
       at android.content.pm.IPackageManager$Stub$Proxy.queryIntentActivities(IPackageManager.java:3949)
       at android.app.ApplicationPackageManager.queryIntentActivitiesAsUser(ApplicationPackageManager.java:815)
       at android.app.ApplicationPackageManager.queryIntentActivities(ApplicationPackageManager.java:807)
       at com.aheaditec.talsec.security.o1.a(SourceFile:153)
       at com.aheaditec.talsec.security.o1.a(SourceFile:141)
       at com.aheaditec.talsec.security.b.a(SourceFile:51)
       at com.aheaditec.talsec.security.o1.c(SourceFile:3)
       at com.aheaditec.talsec.security.o1.a(SourceFile:1)
       at com.aheaditec.talsec.security.a2.d(SourceFile:3)
       at java.lang.Thread.run(Thread.java:818)
```

**Comment by msikyna** [comment]
https://github.com/talsec/Free-RASP-Android/issues/18

**Comment by SirionRazzer** [comment]
This issue is fixed, it will be included in the next release.

**Comment by msikyna** [comment]
Fixed in version #v8.3.0, thank you for reporting it!
